packages/modular-scripts/src/__tests__/app.test.ts

```diff
@@ -78,24 +78,26 @@ describe('when working with a NODE_ENV app', () => {
     expect(tree(path.join(modularRoot, 'dist', 'node-env-app')))
       .toMatchInlineSnapshot(`
       "node-env-app
-      ├─ asset-manifest.json #n1rvuh
+      ├─ asset-manifest.json #1er5q9m
       ├─ favicon.ico #6pu3rg
-      ├─ index.html #1yaenq4
+      ├─ index.html #18m4pzt
       ├─ logo192.png #1nez7vk
       ├─ logo512.png #1hwqvcc
       ├─ manifest.json #19gah8o
       ├─ package.json
       ├─ robots.txt #1sjb8b3
       └─ static
-         └─ js
-            ├─ main.3db228f9.chunk.js #20y3tb
-            ├─ main.3db228f9.chunk.js.map #131rxqt
-            ├─ runtime-main.a0dc6a9b.js #o5bsr9
-            └─ runtime-main.a0dc6a9b.js.map #10n4p35"
+         ├─ js
+         │  ├─ main.6b9113da.js #1jh9hcj
+         │  ├─ main.6b9113da.js.map #2l1ifi
+         │  ├─ runtime-main.182069c4.js #r0hm4v
+         │  └─ runtime-main.182069c4.js.map #ul1xez
+         └─ media
+            └─ javascript,__webpack_public_path__ = __webpack_base_uri__ = htmlWebpackPluginPublicPath;.1feff74f.bin #1327v5u"
```

Comment:
What's this about?

Reply:
@LukeSheard - this is about Webpack 5 controlling the split chunks name via `fileName`, not `chunkFilename`, unlike Webpack 4 - `chunkFilename` is now only used for code splitting. See https://stackoverflow.com/a/67213952 (it's also mentioned in a a comment in this PR's webpack config).

Reply:
Ah, I see now you mean the `media` line. Wow, I've no idea, I'll investigate.

Reply:
@LukeSheard Fixed, this was the cause: https://github.com/jantimon/html-webpack-plugin/issues/1589